### ISSUE TITLE
My web has jitsi meet , if I join jitsi meet wrapper(android) to jitsi meet(web) of this same server. they are joined perfectly but when I unmute audio or video the web has not affected

### ISSUE THREAD

**Issue by wahid8080** [saibotma/jitsi_meet_wrapper#105]

**Comment by dennymoradiya** [comment]
same problem !!

**Comment by saibotma** [comment]
Hey there, thanks for creating the issue. I think I don't really get what you're trying to say. Can you rephrase your problem, please?

**Comment by dennymoradiya** [comment]
i am not able to listen and watch other person even his camera and mice is on.
and it's showing his mice and camera is on at my side , but it's showing black screen at my side.

**Comment by debnath4u** [comment]
I am having same issue, let me try to explain. When Gitsi video not working from Android Mobile Application, so if 2 people doing a group call no one can see their videos, but can listen to each other.
Please help as soon as possible @saibotma

**Comment by NathanCarbon** [comment]
Having the same problem. Not able to see the jitsi webcamera run in chrome in the flutter app

**Comment by saibotma** [comment]
Can you provide me a minimal reproducible example please?

**Comment by wahid8080** [comment]
It was a meeting server problem. It didn't have any jitsi meet plugging problems
